qri/dataset.py

```diff
@@ -134,5 +136,28 @@ def __repr__(self):
         return '[%s]' % (content,)
 
     def _repr_html_(self):
-        content = ', '.join(['%s' % d for d in self])
-        return '<code>[%s]</code>' % (content,)
+        dss_by_user = defaultdict(list)
+        for ds in self:
+            dss_by_user[ds.username].append(ds)
+
+        rows = ""
+        for idx, (username, dss) in enumerate(dss_by_user.items()):
+            for ds in dss:
+                meta_title = ds.versionInfo.meta_title or '(none)'
```

Comment:
Depending on how attached you are to this, we now have a reason to revisit the `versionInfo` question.